packages/next-codemod/bin/transform.ts

```diff
@@ -106,8 +106,9 @@ export async function runTransform(
   if (runInBand) {
     args.push('--run-in-band')
   }
-
-  args.push('--verbose=2')
+  if (verbose) {
+    args.push('--verbose=2')
```

Comment:
should this be default? when there's no verbose it should be 2

Reply:
If `--verbose` not passed, it'll show `ERR` only:

https://github.com/facebook/jscodeshift/blob/f25d7b785784cc2b72e4d6736f3b5ef62cf556a7/src/Runner.js#L50-L63